web/src/lib/components/shared-components/leaflet/marker.svelte

```diff
@@ -37,10 +37,14 @@
   $: if (marker) {
     marker.setLatLng(latlng);
 
-    if (popupContent) {
-      marker.bindPopup(popupContent);
+    if (popupHTML) {
+      marker.bindPopup(popupHTML);
     } else {
       marker.unbindPopup();
     }
   }
 </script>
+
+<span contenteditable="true" bind:innerHTML={popupHTML} class="hide">
+    <slot/>
+</span>
```

Comment:
Here I used a little hack to capture the HTML content of the default slot passed to the Marker component, tell me if there is something more clean than that.

Reply:
This is pretty clever and works for me :+1: